tests/test_utils/test_misc.py

```diff
@@ -160,3 +160,28 @@ def test_import_modules_from_strings():
             ['os.path', '_not_implemented'], allow_failed_imports=True)
         assert imported[0] == osp
         assert imported[1] is None
+
+
+def test_is_method_overriden():
+
+    class Base(object):
+
+        def foo1():
+            pass
+
+        def foo2():
+            pass
+
+    class Sub(Base):
+
+        def foo1():
+            pass
+
+    # test passing sub class directly
+    assert mmcv.is_method_overriden('foo1', Base, Sub)
+    assert not mmcv.is_method_overriden('foo2', Base, Sub)
```

Comment:
Please guard against this example:

![image](https://user-images.githubusercontent.com/9464825/123399743-92a67700-d5d7-11eb-8e31-7893528a8963.png)

Reply:
Should this be an unacceptable situation (trigger AssertionError)?

Reply:
yes

Reply:
Or you may support it. I'm okay with both

Reply:
Done